### PR TITLE
hotfix(production) Read subgraph Urls from .env

### PR DESCRIPTION
**Pull request by mohammadranjbarz**

### CODE REVIEW

**Comment by mohammadranjbarz** [comment]
@mhmdksh After merging this we should add these variables in Vercel for production ENV
* **uniswapV2Subgraph**
* ** honeyswapV2Subgraph**
* **superFluidSubgraph**

**Review by RamRamez — APPROVED** [review]
Thanks @mohammadranjbarz

**Comment by mohammadranjbarz** [comment]
@mhmdksh Feel free to merge it once you added the variables in Vercel panel

File: src/config/development.tsx [hunk 1]
```diff
@@ -184,6 +184,7 @@ const config: EnvConfig = {
 		],
 		regenPools: [],
 		uniswapV2Subgraph:
+			process.env.uniswapV2Subgraph ||
```
Comment (MohammadPCh) [inline src/config/development.tsx:187]: @mohammadranjbarz We have to use `NEXT_PUBLIC_` before the Name to be accessible on the client too.
also please use UPPERCASE for the name for instance: ‍`NEXT_PUBLIC_SUBGRAPH_UNISWAP_V2`

**Review by MohammadPCh — CHANGES_REQUESTED** [review]

File: src/config/development.tsx (hunk 1 above)
Comment (mohammadranjbarz) [reply src/config/development.tsx]: Thanks @MohammadPCh  for your great feedback, done.

**Comment by mohammadranjbarz** [comment]
@mhmdksh 
* **NEXT_PUBLIC_SUBGRAPH_SUPER_FLUID**
* **NEXT_PUBLIC_SUBGRAPH_UNISWAP_V2**
* **NEXT_PUBLIC_SUBGRAPH_HONEYSWAP_V2**

**Comment by mhmdksh** [comment]
> @mhmdksh
> 
> * **NEXT_PUBLIC_SUBGRAPH_SUPER_FLUID**
> * **NEXT_PUBLIC_SUBGRAPH_UNISWAP_V2**
> * **NEXT_PUBLIC_SUBGRAPH_HONEYSWAP_V2**

Thanks for updating the ENV links @mohammadranjbarz
I have setup the ENV variables on our Frontend, this is now safe to merge

**Review by RamRamez — APPROVED** [review]
Thanks @mohammadranjbarz

**Pull request merged**